.circleci/config.yaml

```diff
@@ -0,0 +1,18 @@
+version: 2
+jobs:
+  build_and_test:
+    docker:
+    - {image: 'circleci/golang:1.13'}
```

Comment:
Change to `- image: circleci/golang:1.13`.

Reply:
Take a look at this example: https://github.com/cloudspannerecosystem/harbourbridge/blob/master/.circleci/config.yml